papaparse.js

```diff
@@ -969,7 +987,7 @@
 
 		function parseDynamic(field, value)
 		{
-			if ((_config.dynamicTyping[field] || _config.dynamicTyping) === true)
+			if (_config.dynamicTyping.isIncluded(field))
```

Comment:
I'm wondering if we should create a cache to prevent evaluating the function for each field. This will make parsing faster.